source/GameInterface/Services/Workshops/Patches/WorkshopPropertyPatches.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using GameInterface.Policies;
+using GameInterface.Services.ObjectManager;
+using GameInterface.Services.Workshops.Messages;
+using HarmonyLib;
+using Serilog;
+using System;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.Party;
+using TaleWorlds.CampaignSystem.Settlements.Workshops;
+
+namespace GameInterface.Services.Workshops.Patches
+{
+    public enum PropertyType
+    {
+        Capital,
+        LastRunCampaignTime,
+        WorkshopType
+    }
+
+    [HarmonyPatch(typeof(Workshop))]
+    public class WorkshopPropertyPatches
```

Comment:
We need the following properties synced still.
Also for future reference could you comment in the issue why you are skipping some sync?

- InitialCapital (set in the constructor which requires sync)
- Capital (set in the constructor which requires sync)